### PR TITLE
Unfix matplotlib version

### PR DESCRIPTION
**Pull request by kamo-naoyuki**
I don't know why the matplotlib version is fixed. I opened this PR to check CI tests.

### CODE REVIEW

File: espnet/nets/pytorch_backend/transformer/plot.py [hunk 1]
```diff
@@ -33,12 +33,12 @@ def _plot_and_save_attention(att_w, filename, xtokens=None, ytokens=None):
         ax.yaxis.set_major_locator(MaxNLocator(integer=True))
         # Labels for major ticks
         if xtokens is not None:
-            ax.set_xticks(numpy.linspace(0, len(xtokens) - 1, len(xtokens)))
-            ax.set_xticks(numpy.linspace(0, len(xtokens) - 1, 1), minor=True)
+            ax.set_xticks(numpy.linspace(0, len(xtokens), len(xtokens) + 1))
+            ax.set_xticks(numpy.linspace(0, len(xtokens), 1), minor=True)
```
Comment (kamo-naoyuki) [inline espnet/nets/pytorch_backend/transformer/plot.py:37]: I met following error:

> ValueError: The number of FixedLocator locations (5), usually from a call to set_ticks, does not match the number of ticklabels (6).

`set_xticks` receive  invalid arguments with  the args of `set_xticklabels`: len(xtokens) != len(xtokens != [""])

I think `set_xticks` is, maybe, not required here, but I'm not sure. I simply changed the number to fix this problem.

**Comment by AdityaYadavalli1** [comment]
> I don't know why the matplotlib version is fixed. I opened this PR to check CI tests.

I just happened to make one of the changes in this PR locally and it fixed the [issue](https://github.com/espnet/espnet/issues/4575#issue-1341173693) I raised today. Shall I close that issue?

**Comment by kamo-naoyuki** [comment]
> > I don't know why the matplotlib version is fixed. I opened this PR to check CI tests.
> 
> I just happened to make one of the changes in this PR locally and it fixed the [issue](https://github.com/espnet/espnet/issues/4575#issue-1341173693) I raised today. Shall I close that issue?

Please do as you prefer.

**Comment by kamo-naoyuki** [comment]
Probably, I understood the issues surrounding numpy compatibility.

I think this PR is okay. I'll merge it.

**Pull request merged**